health-sdk/sdk/src/main/res/layout/ghs_fragment_review.xml

```diff
@@ -63,28 +68,39 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/payment_details"
-        style="@style/GiniHealth.PaymentDetailsCard"
-        android:clipToPadding="false"
+        style="@style/GiniHealthTheme.PaymentDetailsCard"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent">
 
+        <View
```

Comment:
[Request] Sorry for the wasted work... We don't need the "drag handle" because the sheet is static ☹️

Reply:
Done, removed it.